README.md

```diff
@@ -1,77 +1,30 @@
-# 1Password Connect
+<p align="center">
+  <img src="https://github.com/1Password/connect/assets/46452606/0f7cf2a8-a290-41fc-b78d-3dfb1017f9be" alt="Header" >
+</p>
 
-The 1Password Connect server provides a REST API that can be used to securely access data from your 1Password account. Learn more about [setting up a Secrets Automation Workflow](https://support.1password.com/secrets-automation/)
+<p align="center">
+  <a href="https://1password.com">
```

Comment:
Looking at the [preview](https://github.com/1Password/connect/blob/2d5b35a9c7d01c02283473dc4c426287f21828e7/README.md#1password-connect) for this README.md file, clicking this link doesn't navigate me to 1password.com (it doesn't actually take me anywhere). I checked Shell Plugins and it doesn't work there either.

Do we want this to be a link to the 1Password homepage? IMO we don't really need to do that and we can just remove it here.
2.

Reply:
The code snippet that I use for my stuff is inspired from [op-go](https://github.com/1Password/op-go/blob/main/README.md) repo:
```markdown
<div style="text-align: center;">
  <img src="assets/connect-illustration.svg" alt="Connect illustration">

  <a href="https://1password.com">
    <h1>1Password Connect</h1>
  </a>
</div>
```